src/api/app/views/webui/request/beta_show_tabs/_review_summary.html.haml

```diff
@@ -0,0 +1,39 @@
+- reviews.each do |review|
+  :ruby
+    review_by = if review.by_user
```

Comment:
We keep doing this over and over again. In this PR alone here and in the [bs_request_overview_avatars_component.html.haml](https://github.com/openSUSE/open-build-service/pull/12863/files#diff-fa44a752f6648ecb8798e93aeeaafd8d77197e4bb05477bc6d6531a55222031b)

I should be able to ask the review if it's `by_project` or `by_package` without having to clear this up (oh it has `by_project`, does it also have a `by_package` maybe?) all the time...